cfgov/unprocessed/css/forms.less

```diff
@@ -607,44 +398,346 @@
+                      <input class="cf-input cf-input_hidden" type="radio" name="cf-input-example_default-inset" id="cf-input-example_default-inset">
+                      <label class="cf-input_label" for="cf-input-example_default-inset">
+                          <span class="cf-input_text">Yes</span>
+                      </label>
+                      <span class="cf-input_radio"></span>
+                  </div>
+              </div>
+              <div class="form-l_col form-l_col-1-2 form-l-inset">
+                  <div class="form-l-inset_container">
+                      <input class="cf-input cf-input_hidden" type="radio" name="cf-input-example_default-inset" id="cf-input-example_default-inset2">
+                      <label class="cf-input_label" for="cf-input-example_default-inset2">
+                          <span class="cf-input_text">No</span>
+                      </label>
+                      <span class="cf-input_radio"></span>
+                  </div>
+              </div>
+          </div>
+        codenotes:
+          - |
+            Structural cheat sheet:
+            -----------------------
+            .form-l-inset
+              .form-l-inset_container
+        notes:
+          - ".form-l-inset_container elements have larger target areas for easier user interaction."
+    tags:
+      - cf-forms
++*/
+
 .form-l-inset {
-    // The default margin between two columns is 30px.
-    // The following rules shrink it to 5px for the insets.
     border-right-width: 2.5px;
     + .form-l-inset { border-left-width: 2.5px; }
     &:last-child { border-right-width: 15px; }
 
-    .form-l-inset_container {
-      padding: unit((@grid_gutter-width / 2) / @base-font-size-px, em)
-               unit((@grid_gutter-width / 2) / @base-font-size-px, em);
-      background: @gray-20;
-    }
+      .form-l-inset_container {
+          background: @input-inset-bg;
+
+          .cf-input_label {
+              margin: 0;
+              padding: unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em )
+                   unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em )
+                   unit( ( @grid_gutter-width / 1.75 ) / @base-font-size-px, em );
+          }
+
+          .cf-input_text {
+              margin-left: unit( @grid_gutter-width / @base-font-size-px, em );
+          }
+
+          .cf-input_radio,
+          .cf-input_checkbox {
+              position: absolute;
+              top: unit( ( @grid_gutter-width / 1.75 ) / @base-font-size-px, em );
+              top: 15px;
+              left: 15px;
+              z-index: 3;
+          }
+
+          &:hover,
+          &:focus,
+          &.focus {
+              .cf-input_checkbox,
+              .cf-input_radio {
+                  border-color: @input-bg;
+                  outline: 1px solid @input-border-focus;
+              }
+              .cf-input_radio {
+                  outline: none;
```

Comment:
Why is this overwriting when it's set on L:713? Also, should we use either outline or box-shadow instead of both throughout these?